src/bci_build/package/package_versions.json

```diff
@@ -44,5 +44,11 @@
         "6": "0.21.2",
         "7": "0.21.2",
         "Tumbleweed": "0.22.1"
+    },
+    "kea":{
+        "6": "2.6.1",
+        "7": "2.6.1",
+        "Tumbleweed": "2.6.1",
+        "version_format": "minor"
     }
-}
\ No newline at end of file
+}
```

Comment:
please run `poetry run update-versions` to re-create this file

Reply:
file is re created.